OoyalaApi.php

```diff
@@ -223,7 +223,6 @@ public function patch($requestPath, $requestBody = array(),
      * @param string $requestPath The path of the resource from the request.
      * @param array  $queryParams The associative array with GET parameters.
      *                            Defaults to array().
-     * @param string $requestBody The POST data to send. Defaults to "".
```

Comment:
This argument is not part of the function definition so removed it from comments.